lib/python/treadmill_aws/hostmanager.py

```diff
@@ -134,7 +152,10 @@ def delete_hosts(ec2_conn, ipa_client, hostnames):
             _LOGGER.debug('Host not found: %s', hostname)
 
     _LOGGER.debug('Delete instances: %r', hostnames)
-    ec2client.delete_instances(ec2_conn, hostnames=hostnames)
+    ec2_chunks = len(hostnames) // 50 + 1
+    for i in range(ec2_chunks):
+        batch = hostnames[i * 50:(i + 1) * 50]
```

Comment:
magic number? Make it _DELETE_BATCH_SIZE and put comment why it is needed.

Reply:
Fixed in subsequent PR.